### ISSUE TITLE
Port reconfiguration not working (deb package)

### ISSUE THREAD

**Issue by thilina27** [NVIDIA/DIGITS#528]
Hi

I have use this command sudo python -m digits.config.edit -v and changed the DIGITS port to 5000 
Then i have configure Data set Configuration 
Its URL is bellow
http://localhost:5000/datasets/images/classification/new

after click on create its redirect to http://localhost/datasets/20160120-104536-a3ef and Browser shows ERR_CONNECTION_REFUSED

Its seems to be not redirecting to port 5000 . Then i change url manually to

http://localhost:5000/datasets/20160120-104536-a3ef Its shows me the output.

**Comment by thilina27** [comment]
Hi

I have use this command sudo python -m digits.config.edit -v  and changed the DIGITS port to 5000 
Then i have configure Data set Configuration 
Its URL is bellow
http://localhost:5000/datasets/images/classification/new

after click on create its redirect to http://localhost/datasets/20160120-104536-a3ef  and Browser shows ERR_CONNECTION_REFUSED

Its seems to be not redirecting to port 5000 . Then i change url manually to 

http://localhost:5000/datasets/20160120-104536-a3ef same error described above comment is shown.

**Comment by thilina27** [comment]
Hi

Now I cant even run DIGITS 2 :disappointed:  

i was trying to run it with ./runme.sh file 
It says address is already in use. (System is just restarted) 

this is my terminal , i have also try to kill DIGITS process but its keep running 


![screenshot from 2016-01-20 11 04 46](https://cloud.githubusercontent.com/assets/13943331/12440798/a93a79c8-bf65-11e5-8de2-38447b2cb1dd.png)

**Comment by thilina27** [comment]
Fixed 1st problem by changing the folder permission. 

Still having the other issues.

**Comment by lukeyeager** [comment]
I'm confused. You can't change the DIGITS port with the command you mentioned: `sudo python -m digits.config.edit -v`.

1. How did you install DIGITS? [With a deb package](https://github.com/NVIDIA/DIGITS/blob/digits-3.0/docs/UbuntuInstall.md)?
2. How did you actually change the port? Did you use `sudo dpkg-reconfigure digits`?

**Comment by thilina27** [comment]
Im really Sorry, pasted wrong command 
sudo dpkg-reconfigure digits this is the one that i have used 

I used this guide to install 

https://github.com/NVIDIA/DIGITS/blob/digits-3.0/docs/UbuntuInstall.md

**Comment by lukeyeager** [comment]
Woah, you're right. 

1. Install the DIGITS deb package
2. Use `sudo dpkg-reconfigure digits` to change the front-end port from 80 to 5000
3. Navigate to `http://localhost:5000/datasets/images/classification/new` and create a dataset
4. Nginx redirects me to `http://localhost/datasets/JOB_ID` (port 80) which gives an error

Thanks for the bug report!

**Comment by lukeyeager** [comment]
It looks like [these two lines](https://github.com/NVIDIA/DIGITS/blob/v3.0.0-rc.3/nginx.site#L9-L11) are the problem.

Try commenting it out from `/etc/nginx/sites-enabled/digits.site` and restarting nginx with `sudo service nginx restart`. That does the trick for me.
```sh
proxy_pass http://127.0.0.1:34448;
#proxy_redirect off;

#proxy_set_header Host $host;
proxy_set_header X-Real-IP $remote_addr;
```

**Comment by lukeyeager** [comment]
Fixed in #530.
